src/Features/CSharp/Portable/Completion/CSharpCompletionOptionsProvider.cs

```diff
@@ -5,16 +5,14 @@
 using System.Composition;
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Options.Providers;
+using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion
 {
     [ExportOptionProvider, Shared]
     internal class CSharpCompletionOptionsProvider : IOptionProvider
```

Comment:
Can this not be removed?  Are we keeping it for compat with people who might still be using something from `CSharpCompletionOptions`?  If so, will that work when there aren't actually any options in it?